README.md

```diff
@@ -110,6 +110,53 @@ Based on the discussion in [#79](https://github.com/js-reporters/js-reporters/is
 
 Additional properties (not defined here) can be added to the Assertion object.
 
+## Details
+
+For details please check out the [docs](https://github.com/js-reporters/js-reporters/blob/add-examples/docs) and especially the [example](https://github.com/js-reporters/js-reporters/blob/add-examples/docs/example.md) which presents a testsuite and its reporting data based on the standard presented above.
+
+For implementations examples please check [Usage of the adapters](#usage-of-the-adapters) and [Integrations](#integrations).
+
+## Usage of the adapters
+
+Listen to the events and receive the emitted data:
+
+```js
+var runner;
+
+// Attach the adapter.
+if (QUnit) {
+  runner = new JsReporters.QUnitAdapter(QUnit);
+} else if (mocha) {
+  runner = new JsReporters.MochaAdapter(mocha);
+} else if (jasmine) {
+  runner = new JsReporters.JasmineAdapter(jasmine.getEnv());
+}
+ 
+ // Listen to the same events for any testing framework.
+ runner.on('testEnd', function(test) {
+   console.log('Test %s has errors:', test.fullname.join(' '), test.errors);
+ });
+ 
+ runner.on('runEnd', function(globalSuite) {
+   var testCounts = globalSuite.testCounts;
+   
+   console.log('Build status: %d', globalSuite.status);
```

Comment:
I was thinking to underline how easily is to get the status for the whole testing, like for continuous integration.

Maybe "Testing status" or "Testsuite status", "Test" sounds a little bit confusing, because you we'll think that is the status only of one test.